LEGO1/mxdirectdraw.cpp

```diff
@@ -189,10 +198,11 @@ void MxDirectDraw::DestroyButNotDirectDraw()
   }
 }
 
-// OFFSET: LEGO1 0x1009E880
+// OFFSET: LEGO1 0x1009E880 STUB
 const char* MxDirectDraw::ErrorToString(HRESULT error)
 {
   //TODO
+  return NULL;
```

Comment:
PR #19 seems to contain a (complete?) implementation of this method. Just wanted to make it explicit to prevent double work.

Reply:
yep, I know. But also in PR #19 this method must be modified to remove any \0 characters at the end of the strings, because it redundant.

Reply:
Hey, I've incorporated this (and other changes) in my PR on your fork. If you accept, I will cancel #19 here and we can merge this one.

Reply:
accepted, now I will recalculate the accuracy